comps/dataprep/pgvector/langchain/prepare_doc_pgvector.py

```diff
@@ -138,5 +211,66 @@ async def ingest_documents(
     raise HTTPException(status_code=400, detail="Must provide either a file or a string list.")
 
 
+@register_microservice(
+    name="opea_service@prepare_doc_pgvector_file", endpoint="/v1/dataprep/get_file", host="0.0.0.0", port=6008
```

Comment:
With PR https://github.com/opea-project/GenAIComps/pull/424, we can use the same port and different endpoint in one microservice. Please unify the port number.

Reply:
Opened new PR https://github.com/opea-project/GenAIComps/pull/444, with the unify the port number.

Closing this due to DCO mess.